icons/flower.svg

```diff
@@ -9,10 +9,14 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M12 7.5a4.5 4.5 0 1 1 4.5 4.5M12 7.5A4.5 4.5 0 1 0 7.5 12M12 7.5V9m-4.5 3a4.5 4.5 0 1 0 4.5 4.5M7.5 12H9m7.5 0a4.5 4.5 0 1 1-4.5 4.5m4.5-4.5H15m-3 4.5V15" />
   <circle cx="12" cy="12" r="3" />
-  <path d="m8 16 1.5-1.5" />
-  <path d="M14.5 9.5 16 8" />
-  <path d="m8 8 1.5 1.5" />
-  <path d="M14.5 14.5 16 16" />
+  <path d="M12 16.5A4.5 4.5 0 1 1 7.5 12 4.5 4.5 0 1 1 12 7.5a4.5 4.5 0 1 1 4.5 4.5 4.5 4.5 0 1 1-4.5 4.5" />
+  <path d="M12 7.5V9" />
+  <path d="M7.5 12H9" />
+  <path d="M16.5 12H15" />
+  <path d="M12 16.5V15" />
+  <path d="m8 16 1.9-1.9" />
+  <path d="M14.1 9.9 16 8" />
+  <path d="m8 8 1.9 1.9" />
+  <path d="M14.1 14.1 16 16" />
```

Comment:
The paths don't meet perfectly, and I think we could easily merge the outside shape as such:
```suggestion
  <path d="M7.5 12A4.5 4.5 0 1 1 12 7.5V9 7.5a4.5 4.5 0 1 1 4.5 4.5H15h1.5a4.5 4.5 0 1 1-4.5 4.5V15v1.5A4.5 4.5 0 1 1 7.5 12H9Z" />
  <path d="m8 8 1.88 1.88" />
  <path d="M14.12 9.88 16 8" />
  <path d="m8 16 1.88-1.88" />
  <path d="M14.12 14.12 16 16" />
```

Reply:
I'm not a fan of overlapping lines, they can produce some black spots when antialiasing is enabled.

My optimizer removes them currently, but I know that we don't really have a guideline on them.

Reply:
Weird, I don't think I've ever encountered this issue, can you provide an example?

I'm fine with keeping them separate, in that case, just the last four diagonal lines need to be fixed:
```suggestion
  <path d="M12 16.5A4.5 4.5 0 1 1 7.5 12 4.5 4.5 0 1 1 12 7.5a4.5 4.5 0 1 1 4.5 4.5 4.5 4.5 0 1 1-4.5 4.5" />
  <path d="M12 7.5V9" />
  <path d="M7.5 12H9" />
  <path d="M16.5 12H15" />
  <path d="M12 16.5V15" />
  <path d="m8 8 1.88 1.88" />
  <path d="M14.12 9.88 16 8" />
  <path d="m8 16 1.88-1.88" />
  <path d="M14.12 14.12 16 16" />
```